resources/views/stories/avatar/single.blade.php

```diff
@@ -0,0 +1,15 @@
+@storybook([
+    'name' => 'Avatar',
```

Comment:
As stories devem ter nomes indicativas do que lá tem dentro. Neste caso, por exemplo:
```suggestion
    'name' => 'Avatar without image',
```